packages/@okta/vuepress-site/docs/guides/add-an-external-idp/main/oktatookta/appidpinokta.md

```diff
@@ -1,18 +1,18 @@
-* **Name** &mdash; enter a name for the Identity Provider configuration.
-* **Client Id** &mdash; paste the client ID that you obtained from the Okta org that represents the Identity Provider in the previous section.
-* **Client Secret** &mdash; paste the secret that you obtained in the previous section.
-* **Scopes** &mdash; leave the defaults. These scopes are included when your Okta org makes a request to the other Okta org that represents the Identity Provider.
+* **Name**: Enter a name for the Identity Provider configuration.
+* **Client Id**: Paste the client ID that you obtained from the Okta org that represents the Identity Provider in the previous section.
+* **Client Secret**: Paste the secret that you obtained in the previous section.
+* **Scopes**: Leave the defaults. These scopes are included when your Okta org makes a request to the other Okta org that represents the Identity Provider.
 
     By default, Okta requires the `email` attribute for a user. The `email` scope is required to create and link the user to Okta's Universal Directory.
 
 In the **Endpoints** section:
 
 Add the following endpoint URLs for the Okta Identity Provider that you are configuring. In the Okta org that represents the Identity Provider, you can find the endpoints in the well-known configuration document (for example, `https://${theOktaIdPOrg}/.well-known/openid-configuration`.
 
-* **Issuer** &mdash; the identifier of the Okta Identity Provider. For example, the Okta org where you created the Identity Provider app: `https://${theOktaIdPOrg}`<br>
-* **Authorization endpoint** &mdash; the URL of the Okta Identity Provider's OAuth 2.0 authorization endpoint. For example: `https://${theOktaIdPOrg}/oauth2/v1/authorize`
-* **Token endpoint** &mdash; the URL of the Okta Identity Provider's token endpoint for obtaining access and ID tokens. For example: `https://${theOktaIdPOrg}/oauth2/v1/token`
-* **JWKS endpoint** &mdash; the URL of the Okta Identity Provider's JSON Web Key Set document. This document contains signing keys that are used to validate the signatures from the provider. For example: `https://${theOktaIdPOrg}/oauth2/v1/keys`
-* **Userinfo endpoint (optional)** &mdash; the endpoint for getting identity information about the user. For example: `https://${theOktaIdPOrg}/oauth2/v1/userinfo`
+* **Issuer**: The identifier of the Okta Identity Provider. For example, the Okta org where you created the Identity Provider app: `https://${theOktaIdPOrg}`<br>
```

Comment:
is <br> required here?

Reply:
@vanngo-okta I don't understand what you are asking

Reply:
lol, it didn't render.. is the `<br>` required after this line?

Reply:
oh probably not - probably left over from something else.I'll remove it!